### ISSUE TITLE
Update title for SEO

### ISSUE THREAD

**Issue by moroshko** [moroshko/accessible-colors#57]
As Mike suggested, we should make our page title more searchable.
It should be up to 64 characters.

**Comment by vedranio** [comment]
"WCAG 2.0 contrast checker that finds closest accessible color combination, given your starting text and background colors"
